summary.md

```diff
@@ -0,0 +1,18 @@
+* [Assumptions](assumptions.md)
+* Drafts
+  * [1 Entrypoint](api-documentation/1.entry-point.md)
+  * [1.1 Security considerations](1.1.security-considerations.md)
+  * [2 API Documentation](api-documentation/2.api-documentation.md)
+    * [2.1 Data structures](api-documentation/2.1.api-documentation-data-structures.md)
+    * [2.2 User document](api-documentation/2.2.api-documentation-user-document.md)
+    * [2.3 Complete API Documentation](api-documentation/2.3.complete-api-documentation.md/2)
```

Comment:
This link is not functional, I guess because of the "/2" at the end.

Reply:
Yep - I've browsed several pages and links were not working in those places.